build-utils/webpack.common.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
       '@Views': path.resolve(__dirname, '../', 'src/views/'),
       '@Utils': path.resolve(__dirname, '../', 'utils/'),
       '@Lib': path.resolve(__dirname, '../', 'src/lib/'),
+      '@Lib': path.resolve(__dirname, '../', 'src/lib/'),
```

Comment:
why is this added again?

Reply:
That was a mistake that wasn't meant to be there, apparently, I already added it before I rebased and didn't notice there was a duplicate. Would change it now.